{
  if (ctx.channel().isActive()) {
    LOG.error("Caught exception",cause);
    sendError(ctx,INTERNAL_SERVER_ERROR);
  }
}
